### DISCUSSION TITLE
Beginner's Guide

### DISCUSSION THREAD

**Discussion by iR00i** [aub-mind/arabert#80] [Q&A]
Hello and Assalamu-alaykum , i am working on my first arabic NLP project, i was wondering if you guys could link me to communities, publications and research regarding Arabic-NLP (specifically for classification).

**Comment by WissamAntoun** [comment]
This is a quick summary of the latest advances in Arabic NLP https://discuss.huggingface.co/t/arabic-nlp-tutorial/3739 

I also encourage you to search and read papers from the workshop on Arabic NLP ( just search WANLP 2021 and older)
